tests/core/test_processor.py

```diff
@@ -643,8 +643,9 @@ async def test_handle_message_with_session_start(
             [{"entity": entity, "start": 6, "end": 22, "value": "Core"}],
         ),
         SlotSet(entity, slot_1[entity]),
+        # DefinePrevUserUtteredFeaturization(False),
         ActionExecuted("utter_greet"),
-        BotUttered("hey there Core!"),
+        BotUttered("hey there Core!"),  # , metadata={"utter_action": "utter_greet"}),
```

Comment:
What's the reason for keeping these comments in here?

Reply:
The comments show how this changes with 2.x

Reply:
Could that be stated somewhere or is this a practice that is in use throughout the code base and I am just unaware of it?

Reply:
You're right, I'll add this to the comment line.

Reply:
I'd vote to remove this 👍🏻